tools/rollup-es3/src/es3/Es3Tokens.ts

```diff
@@ -0,0 +1,169 @@
+            "if (func) { return func(obj, prop); } " +
+            "return undefined; " +
+            "})(%args%)%semi%" 
+    },
+    {
+        funcNames: [/Object\.create[\s]*\(/g ],
+        errorMsg: "[Object.create] is not supported in an ES3 environment, use the helper function CoreUtils.objCreate() or add an explicit existence check",
+        // We are only supporting the version that takes a single argument
+        extract: /(Object)\.(create)[\s]*\(([^,\)]*)\)([;]?)/g,
+        checkGroups: [ 2 ],
+        namedGroups: [
+            { name: "src", idx: 0 },
+            { name: "class", idx: 1 },
+            { name: "function", idx: 2 },
+            { name: "semi", idx: 4},
+            { name: "args", idx: 3}
+        ],
+        replace:"(function(obj) { " +
+                "/* ai_es3_polyfil %function% */ " +
+                "var func = %class%[\"%function%\"]; " +
+                // Use build in Object.create
+                "if (func) { return func(obj); } " +
+                "if (obj == null) { return {}; }; " + 
+                "var type = typeof obj; " +
+                "if (type !== 'object' && type !== 'function') { throw new TypeError('Object prototype may only be an Object:' + obj); } " +
+                "function tmpFunc() {}; tmpFunc.prototype = obj; return new tmpFunc(); " +
+                "})(%args%)%semi%" 
+    },
+    {
+        funcNames: [/Object\.(freeze|seal)[\s]*\(/g ],
+        errorMsg: "[Object.freeze] is not supported in an ES3 environment, use a helper or add an explicit existence check",
+        extract: /(Object)\.(freeze|seal)[\s]*\(([^\)]*)\)([;]?)/g,
+        checkGroups: [ 2 ],
+        namedGroups: [
+            { name: "src", idx: 0 },
+            { name: "class", idx: 1 },
+            { name: "function", idx: 2 },
+            { name: "semi", idx: 4},
+            { name: "args", idx: 3}
+        ],
+        replace:"(function(obj) { " +
+                "/* ai_es3_polyfil %function% */ " +
+                "var func = %class%[\"%function%\"]; " +
+                // Use existing function if available
+                "if (func) { return func(obj); } " +
+                "return obj; " +
+                "})(%args%)%semi%" 
+    }
+];
+
+export const defaultEs3CheckTokens:Array<IEs3CheckKeyword> = [
+    {
+        funcNames: [ /Object\.(defineProperty|create|getPrototypeOf|getOwnPropertyDescriptor|preventExtensions|is|isExtensible|seal|isSealed|freeze|isFrozen|fromEntries|entries|isPrototypeOf|setPrototypeOf)\(/g ],
+        errorMsg: "[%funcName%] is not supported in an ES3 environment, use a helper function or add explicit check for existence"
+    },
+    {
+        funcNames: [ /Object\.(defineProperties)\(/g ],
+        errorMsg: "[%funcName%] is not supported in an ES3 environment, use a helper function or add explicit check for existence",
+        ignoreIds: [ 
+            "applicationinsights-react-js", // Don't break build if these exist in the final react extension
```

Comment:
Ignoring react as the react code and included dependencies uses defineProperties